### DISCUSSION TITLE
Documentation not up to date with router arguments?

### DISCUSSION THREAD

**Discussion by PeterStolz** [fastapi-users/fastapi-users#1219] [General]
In noticed this in the full example where it is:
https://github.com/fastapi-users/fastapi-users/blob/0ad2426a086777de704fa0f5ab0db8c03bb81b80/examples/sqlalchemy/app/app.py#L7-L31
but I had to change most params to get it running like:
```python3
app.include_router(
    fastapi_users.get_auth_router(auth_backend, get_user_manager, authenticator),
    prefix="/auth/jwt",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_register_router(get_user_manager, UserRead, UserCreate),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_reset_password_router(get_user_manager),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_verify_router(get_user_manager, UserRead),
    prefix="/auth",
    tags=["auth"],
)
app.include_router(
    fastapi_users.get_users_router(
        get_user_manager, UserRead, UserUpdate, authenticator
    ),
    prefix="/users",
    tags=["users"],
)
```
Same goes for the detailed Routers subpages
I am on 
```
fastapi-users==11.0.0
fastapi-users-db-sqlalchemy==5.0.0
```
which is the version of the [docs](https://fastapi-users.github.io/fastapi-users/11.0/configuration/full-example/).

Did I miss an abstraction layer or helper that creates the router methods from a local file?

**Comment by frankie567** [comment]
The example shown in the docs is correct. What you probably missed is that `fastapi_users` is an instance of the `FastAPIUsers` class, not the package ( look at the import).

It's defined here: https://github.com/fastapi-users/fastapi-users/blob/master/examples/sqlalchemy/app/users.py#L53

**Reply by PeterStolz** [reply]
Ah yes indeed I overlooked that. Thanks for the quick response
